semeio/jobs/correlated_observations_scaling/obs_utils.py

```diff
@@ -34,14 +35,24 @@ def find_and_expand_wildcards(obs_list, user_dict):
     new_dict = deepcopy(user_dict)
     for main_key, value in user_dict.items():
         new_entries = []
+        non_matching_wildcards = []
         if main_key in ("UPDATE_KEYS", "CALCULATE_KEYS"):
             for val in value["keys"]:
                 if "*" in val["key"]:
-                    new_entries.extend(_expand_wildcard(obs_list, val, val["key"]))
+                    expanded = _expand_wildcard(obs_list, val, val["key"])
+                    if len(expanded) == 0:
+                        non_matching_wildcards.append(val["key"])
```

Comment:
If there is a `non_matching_wildcards` entry shouldn't it trigger `ValidationError` too?

Reply:
I agree. Done!